src/styles/global.scss

```diff
@@ -162,3 +162,14 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+:root {
+  --bc-color-brand: rgb(89,128,34);
```

Comment:
why do you need to re-declare the color? isn't it already declared somewhere in the site's css? is it one of the primary hex colors?